### ISSUE TITLE
=begin and =end comment blocks in a scanned file mess up the line count in report file

### ISSUE THREAD

**Issue by chrisfinne** [presidentbeef/brakeman#628]
If you have a

```ruby
class UserController < ApplicationController
=begin
here is my
multi-line comment
block
=end

```

the line-counts are off when you look in the HTML report and see the highlighted syntax. If you replace the begin/end comment block with normal # comments and re-run the report all looks fine.

**Comment by presidentbeef** [comment]
Hi Chris,

Thanks for reporting this!

It appears ruby_parser just doesn't count those lines at all. I've submitted a potential fix here: https://github.com/seattlerb/ruby_parser/pull/178

**Comment by presidentbeef** [comment]
Fix with ruby_parser 3.6.5: http://blog.zenspider.com/releases/2015/03/ruby_parser-version-3-6-5-has-been-released.html
